build/squire-raw.js

```diff
@@ -1626,16 +1629,12 @@ var firstOrLastLine = function(self){
     var rect = range.getBoundingClientRect()
     var nodeOffset = rect.top
     var parentOffset = parentBlock.getBoundingClientRect().top
-    if(nodeOffset === parentOffset){
-      return {firstLine: true, lastLine: true}
-    }
     nodeOffset = nodeOffset - parentOffset
```

Comment:
"Take into account empty divs for first and last line calculations. Le…
…ft and right arrows also trigger first and last line"